src/LinuxTracing/UprobesUnwindingVisitorTest.cpp

```diff
@@ -644,15 +644,21 @@ TEST_F(UprobesUnwindingVisitorTest,
   PerfEvent{std::move(event)}.Accept(&visitor_);
 
   // On unwinding errors, only the first frame is added to the Callstack.
```

Comment:
This comment no longer holds.

Reply:
And the test name should be changed to "...AndAddressInfos". Same for the test below.

Reply:
Done